src/js/components/Ballot/CandidateItem.jsx

```diff
@@ -108,6 +112,7 @@ class CandidateItem extends Component {
       const twitterDescriptionText = twitterDescription && twitterDescription.length ? `${twitterDescription} ` : '';
       const ballotpediaCandidateSummary = candidate.ballotpedia_candidate_summary;
       let ballotpediaCandidateSummaryText = ballotpediaCandidateSummary && ballotpediaCandidateSummary.length ? ballotpediaCandidateSummary : '';
+      ballotpediaCandidateSummaryText = stripHtmlFromString(ballotpediaCandidateSummaryText);
       ballotpediaCandidateSummaryText = ballotpediaCandidateSummaryText.split(/<[^<>]*>/).join(''); // Strip away any HTML tags
```

Comment:
happy to remove the split or leave it in as an extra precaution

Reply:
@mlevoy any modification to remove the HTML can be put in the `stripHtmlFromString` function. Could you more your `split` code into that function, and search for any other uses of `split` like this so we can consolidate it all in that function?